src/assets/feather.desktop

```diff
@@ -8,4 +8,5 @@ Exec=feather
 Terminal=false
 Categories=Network;
 StartupNotify=false
-StartupWMClass=feather
\ No newline at end of file
+StartupWMClass=feather
++Keywords=crypto;currency;XMR
```

Comment:
Extraneous `+` at the start of the line?

Reply:
I apologize for the rookie mistake.  I copied from a local patch and forgot to remove the extraneous `+`.